README.md

```diff
@@ -660,16 +659,16 @@ and add a suitable font for the small, medium and / or large font, e.g.:
 
 You should use a font name like 'Tahoma' or a path to the desired font.
 
-Any NewGRF file used in a game is stored inside the savegame and will refuse
-to load if you do not have that NewGRF file available. A list of missing files
-can be viewed in the NewGRF window accessible from the file load dialogue window.
+Any NewGRF file used in a game is stored inside the savegame and will refuse to
+load if you do not have that NewGRF file available. A list of missing files can
+be viewed in the NewGRF window accessible from the file load dialogue window.
 
 You can try to obtain the missing files from that NewGRF dialogue or – if they
-are not available online – you can search manually through our [forum's graphics
-development section](http://www.tt-forums.net/viewforum.php?f=66) or GrfCrawler
-(http://grfcrawler.tt-forums.net/). Put the NewGRF files in OpenTTD's newgrf folder
-(see section 4.2 'OpenTTD directories') and rescan the list of available NewGRFs.
-Once you have all missing files, you are set to go.
+are not available online – you can search manually through our
+[forum's graphics development section](https://www.tt-forums.net/viewforum.php?f=66)
+or [GRFCrawler](https://grfcrawler.tt-forums.net). Put the NewGRF files in
```

Comment:
> Also do try to keep to under 80 chars where you can, for consistency.

@LordAro I didn't know what exactly you had in mind, so I did the following changes. What do you think about that?